multisig/proposals/ethereum/vip-021/index.ts

```diff
@@ -0,0 +1,70 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+const CHAINLINK_DAI_FEED = "0xAed0c38402a5d19df6E4c03F4E2DceD6e29c1ee9";
+const CHAINLINK_STALE_PERIOD = 100 * 60;
+
+export const DAI = "0x6B175474E89094C44Da98b954EedeAC495271d0F";
+export const vDAI = "0xd8AdD9B41D4E1cd64Edad8722AB0bA8D35536657";
+export const USDT = "0xdAC17F958D2ee523a2206206994597C13D831ec7";
```

Comment:
Not needed

Reply:
[33a7a2c](https://github.com/VenusProtocol/vips/pull/266/commits/33a7a2ca78d231ebaef71813c4c6985267b33c67)